src/Microsoft.ML.Parquet/ParquetLoader.cs

```diff
@@ -88,48 +89,29 @@ public sealed class Arguments
         internal const string ShortName = "Parquet";
         internal const string ModelSignature = "PARQELDR";
 
+        private const string SchemaCtxName = "Schema.idv";
+
         private readonly IHost _host;
         private readonly Stream _parquetStream;
         private readonly ParquetOptions _parquetOptions;
         private readonly int _columnChunkReadSize;
         private readonly Column[] _columnsLoaded;
-        private readonly DataSet _schemaDataSet;
         private const int _defaultColumnChunkReadSize = 1000000;
 
         private bool _disposed;
+        private long? _rowCount;
 
         private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: ModelSignature,
-                verWrittenCur: 0x00010001, // Initial
+                //verWrittenCur: 0x00010001, // Initial
+                verWrittenCur: 0x00010002, // Add Schema to Model Context
                 verReadableCur: 0x00010001,
```

Comment:
>0x00010001 [](start = 32, length = 10)

Hi Tyler, so is this correct? It may be I just don't understand the scheme. Are you certain that a model written in this latest "2" format *could* be interpreted (if lossily) by the "1" reader that existed? Maybe it is, since it's just another file ...